shoop/admin/modules/products/views/forms.py

```diff
@@ -186,7 +186,20 @@ def __init__(self, **kwargs):
             self.fields["shops"].initial = [default_shop]
 
         self.file_url = self.instance.url
-        self.thumbnail = None
+
+        if self.instance.pk and self.instance.file and isinstance(self.instance.file, Image):
+            try:
+                thumbnail = self.instance.easy_thumbnails_thumbnailer.get_thumbnail({
+                    'size': (64, 64),
+                    'crop': True,
+                    'upscale': True,
+                })
+                self.file_url = self.instance.url
+                self.thumbnail = thumbnail.url
+            except Exception:
+                self.thumbnail = None
```

Comment:
Though, now that I think about it... maybe the _try_get_thumbnail_url free function should be a instance method of ProductMedia?